### PR TITLE
Added comment about file path

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by shadowhand** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/http-interop/http-factory/pull-request/15/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=http-interop/http-factory)

**Comment by Nyholm** [comment]
Thank you for merging.

**Comment by mindplay-dk** [comment]
Please consider the number of times it says "if" in the description of this method.

Every "if" in that description implies an "if" in the implementation, so that's a lot of "ifs", and a lot of explanations and code required to untangle what are essentially different factory-types weaved into one.

This interface is several factories overloaded behind a single API.

In order for implementers to meet those requirements, they will have to write essentially a formalized version of the if/else-ladder described in the comments.

I'm concerned it would be difficult to mock such a dependency for testing.

Is there a case where the client does not _know_ what type of stream they're going to create, or from what?

Note that I do mean that as a question, I'm not being sarcastic - there may be such a case? If there is, that would justify the existence of an overloaded factory method. If there isn't, then it's not a single responsibility, and then, SRP...

**Comment by shadowhand** [comment]
@mindplay-dk your comments should be addressed by #14.

**Comment by Nyholm** [comment]
Good argument @mindplay-dk. We (the code and the arguments) has been back and forth on this issue. Let's gather them all in #14.

File: src/StreamFactoryInterface.php [hunk 1]
```diff
@@ -11,6 +11,9 @@
      *
      * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
      * If the argument is a string, a temporary resource will be created that is writable and seekable.
+     * 
+     * If the arugment is a string it will be interpided as the content of the stream. File names or
```
Comment (harikt) [inline src/StreamFactoryInterface.php:15]: What is the meaning of `interpided` . Is it `Interpreted` ?

**Pull request merged**